scripts/analysis.py

```diff
@@ -126,8 +123,8 @@ def exec_string(list, search, whatwant):
         sqlite query command.
     """
 
-    exec_str = ('select ' + whatwant + ' from resources inner join transactions\
-                on transactions.resourceid = resources.resourceid where '
+    exec_str = ('select ' + whatwant + """ from resources inner join transactions
```

Comment:
I thought it was a good way to have multiple lines of string so I wouldn't need `\`

I'll change it if the `"""` is only for docstrings!